acceptance/bundle/variables/git-branch/script

```diff
@@ -0,0 +1,6 @@
+git-repo-init
+trace $CLI bundle validate -o json | grep -v '"commit"'
+trace $CLI bundle validate
+trace $CLI bundle validate -o json -t dev | grep -v '"commit"'
+trace $CLI bundle validate -t dev | grep -v '"commit"'
```

Comment:
What do the greps do?

Reply:
Remove line matching '"commit"' because it includes hash that varies from run to run.